### ISSUE TITLE
Type mismatch with i18next-fs-backend

### ISSUE THREAD

**Issue by simpautus** [sergiodxa/remix-i18next#103]
### Describe the bug

Using the latest versions of dependencies and following the instructions in readme it gives type error creating the server-side instance in `i18next.server.ts`.

`Property 'type' is missing in type 'typeof I18NexFsBackend' but required in type 'NewableModule<BackendModule<unknown>>'.`

When I'm using version `i18next-fs-backend@1.1.5` I get no error.

### Your Example Website or App

no

### Steps to Reproduce the Bug or Issue

Follow the readme and update dependencies to latest versions.

### Expected behavior

No error.

### Platform

- OS: macOS

```
  "dependencies": {
    "@remix-run/node": "^1.7.5",
    "@remix-run/react": "^1.7.5",
    "@remix-run/serve": "^1.7.5",
    "i18next": "^22.0.4",
    "i18next-browser-languagedetector": "^7.0.1",
    "i18next-fs-backend": "^2.0.0",
    "i18next-http-backend": "^2.0.1",
    "isbot": "^3.5.4",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-i18next": "^12.0.0",
    "remix-i18next": "^4.1.1"
  },
  "devDependencies": {
    "@remix-run/dev": "^1.7.5",
    "@remix-run/eslint-config": "^1.7.5",
    "@types/i18next-fs-backend": "^1.1.2",
    "@types/react": "^18.0.15",
    "@types/react-dom": "^18.0.6",
    "eslint": "^8.23.1",
    "typescript": "^4.7.4"
  },
  "engines": {
    "node": ">=14"
  }
```

**Comment by Davidonium** [comment]
I get the same error when adding remix-i18next following the instructions from the `README.md` on a fresh Indie stack remix project.

**Comment by Davidonium** [comment]
This might be fixable by changing the type of the backend propery:

```typescript
export interface RemixI18NextOption {
    /**
     * The i18next options used to initialize the internal i18next instance.
     */
    i18next?: Omit<InitOptions, "react" | "detection"> | null;
    /**
     * The i18next backend module used to load the translations when creating a
     * new TFunction.
     */
    backend?: Newable<BackendModule<unknown>> & Module; // it was NewableModule<BackendModule<unknown>>
    detection: LanguageDetectorOption;
}
```

In https://github.com/sergiodxa/remix-i18next/blob/main/src/server.ts#L67 
Although the problem looks like a problem from the dependency `i18next`.

**Comment by simpautus** [comment]
> @types/i18next-fs-backend has become redundant

I'm still having the same issue after removing `@types/i18next-fs-backend`.

**Comment by miniplus** [comment]
`@types/i18next-fs-backend` cam be removed from the dependencies if you're using `i18next-fs-backend@^1.2.0 | i18next-fs-backend@^2.0.0` as they now include ts definitions themselves.

The issue is the following, with open PR fix: https://github.com/i18next/i18next-fs-backend/issues/32

**Comment by sergiodxa** [comment]
Looks like this was solved in [i18next-fs-backend@2.0.1](https://github.com/i18next/i18next-fs-backend/releases/tag/v2.0.1).
